app/components/landing-paragraph/template.hbs

```diff
@@ -1,17 +1,20 @@
-<div id="{{get-section-id layout.title}}" class="landing-paragraph-container p-xl" style="{{containerStyle}}">
+<style>
```

Comment:
We should put this in the css file

Reply:
Oh whoops, completely agreed. That's not supposed to be there.